aiida/tools/groups/grouppaths.py

```diff
@@ -0,0 +1,321 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Provides functionality for managing large numbers of AiiDA Groups, via label delimitation."""
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+
```

Comment:
can be removed

Reply:
done in 1e9caab0217ba9eaa43153d7b47a56999d4dfebc